CMakeLists.txt

```diff
@@ -506,6 +506,7 @@ if(WIN32)
         "source/tests/system/nirfmxnr_session_tests.cpp"
         "source/tests/system/nirfmxspecan_driver_api_tests.cpp"
         "source/tests/system/nirfmxspecan_session_tests.cpp"
+        "source/tests/system/nirfmxspecan_restricted_driver_api_tests.cpp"
```

Comment:
Minor: Could you alphabetize this new entry?

Reply:
Fixed in commit 3f1229d.